nn/conformer.py

```diff
@@ -0,0 +1,204 @@
+"""
+Conformer code.
+Ref: https://arxiv.org/abs/2005.08100
+"""
+
+from typing import Tuple, List, Callable, Optional, Dict, Any, Union
+from .. import nn
+import copy
+
+
+class _PositionwiseFeedForward(nn.Module):
```

Comment:
I wonder whether it make sense to have this public, like:
```suggestion
class ConformerPositionwiseFeedForward(nn.Module):
```